config.json

```
@@ -18,7 +18,7 @@
   "fee": 0.25,
   "strategy-name": "MyStrategy",
   "strategies-folder": "strategies",
-  "plots": true,
+  "plots": false,
```

```suggestion
  "plots": true,
```

Don't push the changes to your config file!